src/GenFu/Fillers/DateTimeNullableFillers.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace GenFu
+{
+    public class DateTimeNullableFiller : PropertyFiller<DateTime?>
```

Comment:
I Agree, the code performs the same function, therefore should be delegated. 

I will look into this with the Adapter Pattern but would this not be easier with an Extension Method? For example

DateTime.getRandom(min, max);

I don't like "new" in my code as this kills unit testing design principles (IoC/DI).

I also wanted to implement the solution without altering code that does not require it.